tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,1023 @@
+                    <option value="velocity-curve">Velocity curve</option>
+                    <option value="visibility">Visibility</option>
+                  </param>
+                </when>
+                <when value="custom">
+                  <param name="dataproduct_type" type="text" label="Data product type" help="Logical data product type" />
+                </when>
+              </conditional>
+              <param name="obs_collection" type="text" label="Collection" help="Name of the observation collection" />
+              <param name="obs_title" type="text" label="Observation title (archive specific)" />
+              <param name="obs_id" type="text" label="Observation Id" />
+              <param name="facility_name" type="text" label="Facility name" help="Name of the facility used for this observation" />
+              <param name="instrument_name" type="text" label="Instrument name" help="Name of the instrument used for this observation" />
+              <param name="em_min" type="text" label="Start of the energy range, vacuum wavelength in meters" />
+              <param name="em_max" type="text" label="Stop of the energy range, vacuum wavelength in meters" />
+              <param name="obs_publisher_id" type="text" label="Publisher dataset ID" />
+              <param name="s_fov" type="text" label="Diameter (bounds) of the covered region (deg)" />
+              <param name="calibration_level" type="select" label="Calibration level (0, 1, 2, 3, 4, 5)" >
+                <option value="none">None</option>
+                <option value="0">0</option>
+                <option value="1">1</option>
+                <option value="2">2</option>
+                <option value="3">3</option>
+                <option value="4">4</option>
+                <option value="5">5</option>
+              </param>
+              <param name="order_by" type="select" label="Order by" help="Field to use for ordering the results">
+                <option value="none" selected="true">No specific order</option>
+                <option value="size" >Order by size (Ko)</option>
+                <option value="collection" >Order by collection</option>
+                <option value="object" >Order by Target Name, alphabetically</option>
+              </param>
+            </when>
+            <when value="raw_query">
+              <param name="table" type="text" label="Table name" help="Name of the table you want to query (FROM clause)" />
+              <section name="where_clause" title="WHERE clause (field = value)" expanded="false">
+                <param name="where_field" type="text" label="WHERE field" help="WHERE **field** = value" />
+                <param name="where_condition" type="text" label="WHERE value" help="WHERE field = **value**" />
+              </section>
+              <param name="url_field" type="text" label="Url field" help="Table field containing the url of the file to download" />
```

Comment:
does this need https:// or not?

Reply:
It's name of the table column which contains URLs. Name of the column itself is a string with no http://

Reply:
Yes it does. Since its a table field. I assume one ASCII is allowed, not whitespaces etc? If so we could restrict the input field here any in other places.